### ISSUE TITLE
Scaling of Objects in MeshLibrary.

### ISSUE THREAD

**Issue by sairam4123** [godotengine/godot-proposals#1617]
**Describe the project you are working on:** City Building Game

**Describe the problem or limitation you are having in your project:** Object's scaling is reset when you convert a scene into a MeshLibrary.

**Describe the feature / enhancement and how it helps to overcome the problem or limitation:** Support scaling in Meshlib.

**Describe how your proposal will work, with code, pseudocode, mockups, and/or diagrams:** I don't have one right now, I'll make one.

**If this enhancement will not be used often, can it be worked around with a few lines of script?:** I've not researched about this. But I think, there's no way to workaround this with GDScript, there might be a way to workaround this issue with GDNative tho.

**Is there a reason why this should be core and not an add-on in the asset library?:** This is a little feature, but a important one that saves a lot of time for the developers.

**Comment by aaronfranke** [comment]
Makes sense. However, this would make the most sense to at the level of creating the MeshLibrary. This way MeshLibrary doesn't need to know about scaled meshes, only the tool that converts scenes to MeshLibrary needs to know about it.
